api/include/can_protocols/steering_can_protocol.h

```diff
@@ -13,29 +13,36 @@
 #include "magic.h"
 
 
+/*
+ * @brief CAN ID representing the range of steering messages.
+ *
+ */
+#define OSCC_STEERING_CAN_ID_INDEX (0x60)
+
+
 /*
  * @brief Steering enable message (CAN frame) ID.
  *
  */
-#define OSCC_STEERING_ENABLE_CAN_ID (0x54)
+#define OSCC_STEERING_ENABLE_CAN_ID (0x60)
```

Comment:
This overlaps with the old OSCC_BRAKE_COMMAND_CAN_ID. If a DriveKit is flashed with v1.1.1 and oscc-joystick-commander is built and run against this branch, joystick-commander brake commands will be interpreted as steering enable commands which will cause the spoof relay to chatter.